simulator/docker/prepare-docker.sh

```diff
@@ -7,26 +7,27 @@ echo "🚀 Preparing Docker build environment..."
 
 # Get the script's directory
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-PROJECT_ROOT="$SCRIPT_DIR/../.."
-
-# Build new distribution
-echo "📦 Building simulator distribution..."
-cd "$PROJECT_ROOT"
-./gradlew :simulator:assemble
+PROJECT_ROOT="$SCRIPT_DIR/../../.."
```

Comment:
Again, here we are going out of the build root `after` we have done our initial setup of copying all needed files. The execution itself also must not go out of the build root, everything that is needed for the execution must be present and accessible inside the build root.

Reply:
Removed this script.